src/components/select-box/select-box.njk

```diff
@@ -0,0 +1,43 @@
+{% from "select-box/macro.njk" import govukSelectBox %}
+
+
+{{ govukSelectBox(
+  id='select-box-1',
+  name='select-box-1',
+  options=[
+    {
+      value: '1',
+      label: 'GOV.UK frontend option 1'
+    },
+    {
+      value: '2',
+      label: 'GOV.UK frontend option 2'
+    },
+    {
+      value: '3',
+      label: 'GOV.UK frontend option 3'
+    }
+  ]
+)}}
+
+{{ govukSelectBox(
+  hasLabelWithText='Default label text',
+  labelClasses='select-box__label',
```

Comment:
if you want to render a label with the select box thas has addidional custom class then you'd pass these two variables. I can put them at then end